CoreScriptsRoot/Modules/VR/NotificationHub.lua

```diff
@@ -425,7 +446,9 @@ end
 NotificationGroup.new("Friends",        "Friends",  1)
 NotificationGroup.new("BadgeAwards",    "Badges",   2)
 NotificationGroup.new("PlayerPoints",   "Points",   3)
-NotificationGroup.new("Developer",      "Other",    4)
+
+--Don't want this right now.
+--NotificationGroup.new("Developer",      "Other",    4)
```

Comment:
For this kind of thing what I did in VR keyboard (for voice chat) was create a function at the top `IsDeveloperGroupEnabled()` function and just return false for now. Then when we want it we can turn make it true at a later point.